### ISSUE TITLE
[intelliJ][Report to Microsoft][Spark on Arcadia]Get error when run interactive console with empty computes.

### ISSUE THREAD

**Issue by v-jiche** [microsoft/azure-tools-for-java#3508]
Build:
azure-toolkit-for-intellij-2019.2.develop.1234.09-05-2019

Repro Steps:
1. Create an Arcadia configuration, do not choose value.
(Correct/fault/empty main class + empty spark computes = error message)
![image](https://user-images.githubusercontent.com/35950097/64503813-6f75df00-d2ff-11e9-9e90-e657f93ab28b.png)
2. Run interactive console from Tools/Spark console

Result:
Get error.
![image](https://user-images.githubusercontent.com/35950097/64503847-9502e880-d2ff-11e9-87a4-cfd82683ab0d.png)

Expectation:
![image](https://user-images.githubusercontent.com/35950097/64503856-a2b86e00-d2ff-11e9-9467-2a7aaf04504e.png)

```
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.microsoft.azure.hdinsight.spark.console.RunSparkConsoleActionDelegate.actionPerformed(RunSparkConsoleActionDelegate.kt:59)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:265)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:282)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.lambda$actionPerformed$0(ActionMenuItem.java:292)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:265)
	at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.runOnOwnContext(IdeFocusManagerImpl.java:107)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.actionPerformed(ActionMenuItem.java:282)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.lambda$fireActionPerformed$0(ActionMenuItem.java:111)
	at com.intellij.openapi.application.TransactionGuardImpl.runSyncTransaction(TransactionGuardImpl.java:82)
	at com.intellij.openapi.application.TransactionGuardImpl.lambda$submitTransaction$1(TransactionGuardImpl.java:106)
	at com.intellij.openapi.application.TransactionGuardImpl.submitTransaction(TransactionGuardImpl.java:115)
	at com.intellij.openapi.application.TransactionGuard.submitTransaction(TransactionGuard.java:121)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:111)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.JToggleButton$ToggleButtonModel.setPressed(JToggleButton.java:401)
	at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:369)
	at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:349)
	at com.intellij.openapi.wm.impl.GlobalMenuLinux.lambda$_handleEvent$13(GlobalMenuLinux.java:698)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:312)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.doRun(LaterInvocator.java:433)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.runNextEvent(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:878)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:827)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:466)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:704)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:465)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
Caused by: kotlin.UninitializedPropertyAccessException: lateinit property tenantId has not been initialized
	at com.microsoft.azure.hdinsight.spark.run.configuration.ArcadiaSparkSubmitModel.getTenantId(ArcadiaSparkSubmitModel.kt:37)
	at com.microsoft.azure.projectarcadia.spark.console.ArcadiaSparkScalaLivyConsoleRunConfiguration.checkRunnerSettings(ArcadiaSparkScalaLivyConsoleRunConfiguration.kt:71)
	at com.microsoft.azure.hdinsight.spark.run.action.RunConfigurationActionUtils.checkRunnerSettings(RunConfigurationActionUtils.kt:87)
	at com.microsoft.azure.hdinsight.spark.run.action.RunConfigurationActionUtils.getRunConfigurationError(RunConfigurationActionUtils.kt:92)
	at com.microsoft.azure.hdinsight.spark.run.action.RunConfigurationActionUtils.runEnvironmentProfileWithCheckSettings(RunConfigurationActionUtils.kt:53)
	at com.microsoft.azure.hdinsight.spark.console.RunSparkScalaConsoleAction.runFromSetting(RunSparkScalaConsoleAction.kt:144)
	at com.microsoft.azure.hdinsight.spark.console.RunSparkScalaConsoleAction.access$runFromSetting(RunSparkScalaConsoleAction.kt:52)
	at com.microsoft.azure.hdinsight.spark.console.RunSparkScalaConsoleAction$runExisting$1.invoke(RunSparkScalaConsoleAction.kt:119)
	at com.microsoft.azure.hdinsight.spark.console.RunSparkScalaConsoleAction$runExisting$1.invoke(RunSparkScalaConsoleAction.kt:52)
	at com.microsoft.intellij.util.ApplicationHelpersKt.runInReadAction(ApplicationHelpers.kt:42)
	at com.microsoft.azure.hdinsight.spark.console.RunSparkScalaConsoleAction.runExisting(RunSparkScalaConsoleAction.kt:118)
	at com.microsoft.azure.hdinsight.spark.console.RunSparkScalaConsoleAction.onActionPerformed(RunSparkScalaConsoleAction.kt:71)
	at com.microsoft.azuretools.ijidea.utility.AzureAnAction.actionPerformed(AzureAnAction.java:75)
	... 46 more
```

**Comment by t-rufang** [comment]
Fixed in #3512

**Comment by v-jiche** [comment]
Verified. Fixed.
Build: develop 1241
